audit-service/pom.xml

```diff
@@ -163,6 +163,12 @@
             <version>${alpakka.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>org.assertj</groupId>
+          <artifactId>assertj-guava</artifactId>
+          <version>3.3.0</version>
+          <scope>test</scope>
+        </dependency>
     </dependencies>
```

Comment:
```suggestion
        </dependency>
    </dependencies>
```

Reply:
Do'n think that we need this library for the existing tests.

Reply:
It's a shame, but we cannot exclude JUnit4 as Testcontainers relies on it's Rules implementation which are not in JUnit5.